app/assets/stylesheets/layouts/footer.scss

```diff
@@ -0,0 +1,31 @@
+footer {
+  position: relative;
+
+  .row {
```

Comment:
do we need this on the row? Is it ok to do it on footer?

Reply:
I concur. oops.😬